src/Tokenizer/TokensAnalyzer.php

```diff
@@ -181,6 +178,8 @@ public function isBlockMultiline(Tokens $tokens, int $index): bool
      * 'final'      bool
      *
      * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array{visibility: ?int, static: bool, abstract: bool, final: bool}
```

Comment:
PhpStorm [added](https://blog.jetbrains.com/phpstorm/2022/02/phpstorm-2022-1-eap-3/) support for this syntax in latest EAP :)

<img width="437" alt="image" src="https://user-images.githubusercontent.com/26096713/155888059-01e3c9fb-05a2-4d11-9ed9-d9f62ef69620.png">